src/stores/store.jsx

```diff
@@ -1328,6 +1320,115 @@ class Store {
     })
   }
 
+  getCoverageHoldings = async () => {
+    const account = store.getStore('account')
+    const protektContracts = store.getStore('protektContracts')
+
+    if(!account || !account.address) {
+      return false
+    }
+
+    const web3 = await this._getWeb3Provider();
+    if(!web3) {
+      return null
+    }
+
+    async.times(protektContracts.length, (index, callback) => {
+      async.parallel([
+        (callbackInner) => { this._getERC20Balance(web3, protektContracts[index].pTokenAddress, account, callbackInner) },
+        (callbackInner) => { this._getTokenUSDPrice(web3, protektContracts[index].pTokenAddress, protektContracts[index].pTokenContractABI, account, callbackInner) },
```

Comment:
So I believe this should be the price of the underlyingToken. CoinGecko isn't going to have a price for the pToken but will for cDAI (in the Compound-DAI example.

Then you have:
pTokenBalance
User's share of underlyingToken = pTokenBalance * getPricePerFullShare()
amountCoveredUsd = User's share of underlyingToken * Price of underlyingToken